app/views/licences/tabs/set-up.njk

```diff
@@ -10,135 +10,215 @@
   {% endfor %}
 {% endmacro %}
 
-{% set chargeVersionRows = [] %}
-
-{% set chargeVersionHead = [
-  {
-    text: "Start date"
-  },
-  {
-    text: "End date"
-  },
-  {
-    text: "Reason"
-  },
-  {
-    text: "Status"
-  },
-  {
-    text: "Action"
-  }
-] %}
-
-{% for chargeVersion in  chargeInformation %}
-  {% set chargeVersionRows = (chargeVersionRows.push([
+<h2 class="govuk-heading-l">Licence set up</h2>
+
+{% if enableRequirementsForReturns %}
+  <h3 class="govuk-heading-m"> Requirements for returns </h3>
+
+  {% if returnVersions.length == 0 %}
+    <p class="govuk-body"> No requirements for returns for this licence.</p>
+  {% endif %}
+
+  {% if returnVersions.length > 0 %}
+    {% set returnVersionsTableRows = [] %}
+
+    {% set returnVersionsTableHead = [
+      {
+        text: "Start date"
+      },
+      {
+        text: "End date"
+      },
+      {
+        text: "Reason"
+      },
+      {
+        text: "Status"
+      },
+      {
+        text: "Action"
+      }
+    ] %}
```

Comment:
Do we need to set these separately? Or could we just include them when we define the table below?

(And yes, I've just clocked the other tables were built this way and I approved the previous PRs 🤦 . But I don't think they have to be. IMHO I think it's simpler to just define them when declaring the table.)